resources-domain/resources-service/scripts/elasticsearch/create-index.ts

```diff
@@ -19,9 +19,30 @@ import {
   RESOURCE_INDEX_TYPE,
   resourceIndexConfiguration,
 } from '@tech-matters/resources-search-config';
+import { STS } from 'aws-sdk';
 
 const shortCode = process.argv[2] || 'as';
+const accountSid =
+  shortCode.length === 34 && shortCode.toUpperCase().startsWith('AC')
+    ? shortCode
+    : undefined;
```

Comment:
Backed out that change - I was trying to avoid having to call SSM but you still need to even when you don't look up the Account SID so it doesn't help - removed